packages/@textlint/config-loader/src/config-loader.ts

```diff
@@ -201,6 +203,13 @@ export const loadConfig = async (options: TextlintConfigLoaderOptions): Promise<
  */
 export const loadRawConfig = async (options: TextlintConfigLoaderOptions): Promise<TextlintConfigLoaderRawResult> => {
     try {
+        if (
+            options.configFilePath &&
+            fs.existsSync(options.configFilePath) &&
+            !isUtf8(fs.readFileSync(options.configFilePath))
+        ) {
+            throw new Error("textlint configuration file must be encoded in UTF-8");
+        }
         const results = rcFile<TextlintRcConfig>("textlint", {
```

Comment:
`rcFile` will return actual config file path.
Therefore, we may write the following.

```ts
const results = rcFile<TextlintRcConfig>("textlint",  ... );
if (!result) { ... }
assertConfigFileIsUtf8(results.filePath);
```